README.md

```diff
@@ -1,75 +1,159 @@
+
+- Displays a random greeting
+- Consumes an api created with RoR
+- Uses two apps to create a full-stack application, fron-end and backend separate
+
+### Key Features <a name="key-features"></a>
+
+- N/A
+
+### Related-repos <a name="Related-repos"></a>
+
+- https://github.com/Loltolo-Lesapiti/-hello-rails-back-end.git - Hello rails backend.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
+In order to run this project you need: Git and Node.js installed on your computer.
+
+- For windows users you can download Git from here: https://git-scm.com/download/win and Node.js from here: https://nodejs.org/en/download/
+- For Mac users you can download Git from here: https://git-scm.com/download/mac and Node.js from here: https://nodejs.org/en/download/
+- For Linux users you can download Git from here: https://git-scm.com/download/linux and Node.js from here: https://nodejs.org/en/download/
+
 ### Setup
 
+- Clone this repository to your desired folder:
+
+`git clone https://github.com/Loltolo-Lesapiti/-hello-rails-back-end.git`
+
+`cd your-folder`
+
+- npm install # to install all the dependencies
+
 ### Install
 
-### Usage
+Install this project with:
 
-### Run tests
+- cd your-folder
+- npm install
 
-### Deployment
+### Usage
 
+To run the project, execute the following command:
 
+- npm start # for running the project in development mode
+- npm run build # for building the project for production
 
-## Authors
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-👤 **Author1**
+## 👥 Authors <a name="authors"></a>
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Petro Loltolo Lesapiti**
 
-👤 **Author2**
+- GitHub: [@petrolesapiti](https://github.com/Loltolo-Lesapiti)
+- LinkedIn: [@petrolesapiti](https://www.linkedin.com/in/petrolesapitiloltolo/)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🤝 Contributing
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
 
-Give a ⭐️ if you like this project!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Acknowledgments
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-- Hat tip to anyone whose code was used
```

Comment:
- [OPTIONAL] Let's make sure to fill the `Show your support and ` Acknowledgments` sections. This will make your README descriptive. 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/233806612-1177197a-b02e-4126-8c9c-a5687b9f0b52.png)